src/components/forms/account/AccountForm.tsx

```diff
@@ -158,6 +158,7 @@ export default function AccountForm({
                 isClearable={false}
                 onlyPlaceholders
                 placeholder="<parent account>"
+                ignoreAccounts={[form.getValues('guid')]}
```

Comment:
I've changed the purpose of `ignoreAccounts`. Initially it was to filter out types of accounts but the name is confusing and wasn't being used. Now it's with a list of guid.

This fixes so an account can't be it's own parent